packages/modular-scripts/react-dev-utils/logger.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const isCI = require('is-ci');
+const chalk = require('chalk');
+
+const PREFIX = '[modular] ';
+
+const DEBUG =
+  process.env.MODULAR_LOGGER_DEBUG || process.argv.includes('--verbose');
+const SILENT = process.env.MODULAR_LOGGER_MUTE;
+const isInteractive = process.stdout.isTTY;
+
+function printStdErr(x) {
+  if (SILENT) {
+    return;
+  }
+  x.split('\n').forEach((l) => {
```

Comment:
Wouldn't `.split(/\r?\n/)` be more universal?

Reply:
Yeah - when I wrote this initially in the main `modular-scripts` package I didn't consider it - I'll update both in place.